lib/hacluster.pm

```diff
@@ -1098,7 +1098,7 @@ sub pre_run_hook {
 sub post_run_hook {
     my ($self) = @_;
 
-    record_avc_selinux_alerts() if is_sle('16+');
+    $self->record_avc_selinux_alerts() if is_sle('16+');
```

Comment:
Please use `has_selinux()` for HA and SAP files.

Reply:
So can we unblock this?

Reply:
Yes sorry for the delay.